<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileAttributeSettingDialog</class>
 <widget class="QDialog" name="FileAttributeSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="title">
        <string>オーナー</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_OwnerRead">
          <property name="text">
           <string>呼出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_OwnerWrite">
          <property name="text">
           <string>書込</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_OwnerExec">
          <property name="text">
           <string>実行</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>グループ</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_GroupRead">
          <property name="text">
           <string>呼出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_GroupWrite">
          <property name="text">
           <string>書込</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_GroupExec">
          <property name="text">
           <string>実行</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>その他</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_OtherRead">
          <property name="text">
           <string>呼出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_OtherWrite">
          <property name="text">
           <string>書込</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_OtherExec">
          <property name="text">
           <string>実行</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>現在の属性</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_CurrentAttr">
          <property name="maxLength">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Help|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>helpRequested()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_OwnerRead</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_OwnerRead()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_OtherWrite</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_OtherWrite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_OwnerExec</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_OwnerExec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_GroupRead</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_GroupRead()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_GroupWrite</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_GroupWrite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_GroupExec</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_GroupExec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_OtherRead</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_OtherRead()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_OwnerWrite</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_OwnerWrite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_OtherExec</sender>
   <signal>clicked()</signal>
   <receiver>FileAttributeSettingDialog</receiver>
   <slot>onClick_checkBox_OtherExec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>help()</slot>
  <slot>onClick_checkBox_OwnerRead()</slot>
  <slot>onClick_checkBox_OtherWrite()</slot>
  <slot>onClick_checkBox_OwnerExec()</slot>
  <slot>onClick_checkBox_GroupRead()</slot>
  <slot>onClick_checkBox_GroupWrite()</slot>
  <slot>onClick_checkBox_GroupExec()</slot>
  <slot>onClick_checkBox_OtherRead()</slot>
  <slot>onClick_checkBox_OwnerWrite()</slot>
  <slot>onClick_checkBox_OtherExec()</slot>
 </slots>
</ui>
